core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -2,27 +2,14 @@ package org.broadinstitute.dsde.workbench.leonardo.http
 
 import org.broadinstitute.dsde.workbench.google2.DiskName
 import org.broadinstitute.dsde.workbench.google2.KubernetesSerializableName.ServiceName
-import org.broadinstitute.dsde.workbench.leonardo.{
-  App,
-  AppError,
-  AppName,
-  AppStatus,
-  AppType,
-  AuditInfo,
-  CloudContext,
-  CloudProvider,
-  KubernetesCluster,
-  KubernetesRuntimeConfig,
-  LabelMap,
-  Nodepool,
-  WorkspaceId
-}
+import org.broadinstitute.dsde.workbench.leonardo.{App, AppAccessScope, AppError, AppName, AppStatus, AppType, AuditInfo, CloudContext, CloudProvider, KubernetesCluster, KubernetesRuntimeConfig, LabelMap, Nodepool, WorkspaceId}
 import org.http4s.Uri
 
 import java.net.URL
 
 final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRuntimeConfig],
                                   appType: AppType,
+                                  accessScope: Option[AppAccessScope],
```

Comment:
Since `CreateAppRequest` is shared between the V1 and V2 endpoints, I chose to keep `accessScope` optional until the Sam V2 Resource creation

Reply:
Makes sense - we should default to the existing behavior (of private apps) if `accessScope` is not passed.